src/procedures/utils/api-key.js

```diff
@@ -93,6 +93,18 @@ module.exports.AlphaVantageKey = new ApiKey(
     'ALPHA_VANTAGE_KEY'
 );
 
+module.exports.TheCatApiKey = new ApiKey(
+    'TheCatApi',
+    'https://thecatapi.com/signup',
+    'THE_CAT_API_KEY'
```

Comment:
This doesn't need to be set (unless there is a reason it needs this env name). This feature was initially added for backwards compatibility; the default should be good for most cases.